nixos/modules/services/networking/ssh/ssh3-server.nix

```diff
@@ -0,0 +1,94 @@
+{ pkgs, config, lib, ... }:
+let
+  inherit (lib) mkEnableOption mkOption mkPackageOptionMD mkIf types;
+  cfg = config.services.ssh3-server;
+  acmeDirectory = config.security.acme.certs."${cfg.domain}".directory;
+in
+  {
+    options.services.ssh3-server = {
+      enable = mkEnableOption "ssh3-server, a SSH3 (QUIC+TLS 1.3) server for secure remote logins";
+
+      package = mkPackageOptionMD pkgs "ssh3" { };
+      listenAddresses = mkOption {
+        type = with types; listOf (submodule {
+          options = {
+            addr = mkOption {
+              type = types.str;
+              description = lib.mdDoc ''
+                Host, IPv4 or IPv6 address to listen to.
+              '';
+            };
+            port = mkOption {
+              type = types.int;
+              description = lib.mdDoc ''
+                Port to listen to.
+              '';
+            };
+          };
+        });
+        # Keep 22 or 443... ?
+        default = [ { addr = "[::]"; port = 443; } ];
+        example = [ { addr = "192.168.3.1"; port = 22; } { addr = "0.0.0.0"; port = 64022; } ];
+        description = lib.mdDoc ''
```

Comment:
Isn't `lib.mdDoc` the identity function ?

Reply:
Done.